src/settings/versions/v1/v1.ts

```diff
@@ -83,6 +83,12 @@ export const settingsSchema = z.object({
         .filter(s => !isValidIgnorePattern(s)).length === 0,
         {message: "Invalid ignore pattern"}
     ),
+    ignoredTags: z.string().refine((value) => value
+        .split(/[\s,]+/)
+        .filter(s => s.trim().length > 0)
+        .filter(s => !s.startsWith("#") || s.includes(" ")).length === 0,
+        {message: "Invalid tag"}
```

Comment:
The error messages are shown to the user. Can you maybe make them more descriptive? E.g., cannot have spaces, needs `#,` etc. You can do this by making this a multi-line function with multiple returns, each with different message depending on the issue

Reply:
After removing the requirement for a # prefix, I'm not actually sure what would constitute an invalid tag. Would it be fine to have no checks here at all?

(The user can already split their tag list on spaces if they want)

Reply:
I would still check for the space since tags cannot have spaces. I only just released the settings that say: `... Enter one tag per line or a comma-separated list.` Maybe we should keep this consistent with the ignore file paths and do it only with one tag per line. Then, the space check makes sense again. These checks then mainly check if the formatting is consistent with how we expect it. If not, we give them an easy to understand instructions via the `message` on how to fix it or what is wrong. Make sense?

Reply:
I think the code should be able to handle any input here - if the tag doesn't exist, it just won't match, so there's nothing to validate. The current solution lets users input tags in pretty much any way that makes sense to them, which I think is helpful with all the different ways various applications and sites handle them.

For example, currently all of these will work:

`#secret #private #secure`
`secret private secure`
`#secret, #private, #secure`
`#secret, private secure`
```
secret
private, secure
```

The only thing I think we could do to improve that is tell them that "tag (tag name) does not exist)", if we want to go that far.

Reply:
No, I don't want to go to validate if the tag exists. That is too much hassle, and like you said, they can always add it in the future.

I mainly want consistency in the settings, such that:
- the way of specifying is (roughly) the same as how we do it for the paths.
- there is one obvious way to do it. Supporting multiple ways always becomes hard to maintain with backward compatibility over time. So either they all have `#` or none of them do, and either the separator is `\n` or a, `.`. I personally, prefer the no `#` and a `\n` seperator route.

I mainly want to keep support thing the gitignore mental model in the way we input stuff

Reply:
No worries! Sounds fair enough.